dev-requirements.txt

```diff
@@ -1,12 +1,11 @@
 # Pip requirements file for development dependencies.
 
 
-# Include install dependencies
--r requirements.txt
-
-
 # Direct dependencies for development and indirect dependencies for development
-# that are needed for some reason (must be consistent with minimum-constraints.txt)
+# that are needed for some reason (must be consistent with minimum-constraints-develop.txt)
```

Comment:
above line can say Direct and indirect dependencies for development . . .

Reply:
These are all direct dependencies but only some indirect dependencies (those that are needed for some reason). So just shortening the sentence as proposed does not make the distinction between all and some. But maybe the original sentence also did not do a good job in distinguishing that.

Changed it to:
```
Direct and necessary indirect dependencies for development (...)
```